test/server/cli-argument-parser-test.js

```diff
@@ -644,6 +644,51 @@ describe('CLI argument parser', function () {
             });
     });
 
+    describe('Quarantine Options', function () {
```

Comment:
Please add some functional test for this feature. We already implemented some functional tests in quarantine. You can take a look on it:
https://github.com/DevExpress/testcafe/blob/e154ecfb5d40ebc92a32ceab87e82a335701d5bd/test/functional/fixtures/api/es-next/take-screenshot/testcafe-fixtures/take-screenshot.js#L58

Reply:
Thanks for the suggestion. I'm getting an error when trying to run the functional tests locally (`Error: Unable to find the browser. "ie" is not a browser alias or path to an executable file.`).

Is there a way I can trigger these without IE being installed?

Reply:
Do you mean tasks in gulpfile.js?
Yes, we have tasks to run tests without IE: https://github.com/DevExpress/testcafe/blob/e3c1fea1e1631777ed8a643503da50ca397b70a9/Gulpfile.js#L824

Reply:
Ahh great, I didn't see that one 🤦🏻‍♂️. Thanks. I'll push in my changes later today.